### PR TITLE
Change `Contact` and `AddCommand` classes to abstract

### PR DESCRIPTION
**Pull request by McNaBry**

### CODE REVIEW

**Comment by McNaBry** [comment]
Closes #74

File: src/test/java/seedu/address/model/contact/ContactTest.java
Comment (CJ-Lee01) [inline src/test/java/seedu/address/model/contact/ContactTest.java:1]: is this file necessary?

File: src/test/java/seedu/address/testutil/TypicalContacts.java [hunk 1]
```diff
@@ -149,32 +87,20 @@ public class TypicalContacts {
             .build();
 
     // Manually added
-    public static final Contact HOON = new ContactBuilder()
-            .withName("Hoon Meier")
-            .withId("test_8-123")
-            .withPhone("8482424")
-            .withEmail("stefan@example.com")
-            .withAddress("little india")
+    public static final Organization TIKTOK = new OrganizationBuilder()
```
Comment (CJ-Lee01) [inline src/test/java/seedu/address/testutil/TypicalContacts.java:90]: Not really sure if this way of testing is ideal but I will close one eye since there's the cleanup phase

Comment (CJ-Lee01) [reply src/test/java/seedu/address/testutil/TypicalContacts.java:90]: The one where we declare `VALID_NAME` sth like that

**Review by CJ-Lee01 — APPROVED** [review]
Seems good to me.

File: src/test/java/seedu/address/testutil/TypicalContacts.java (hunk 1 above)
Comment (McNaBry) [reply src/test/java/seedu/address/testutil/TypicalContacts.java:90]: With the linking between organization and recruiter, its a bit messy to test but yeah should look into cleaning things up and making sure the dependencies for each test do not mess things up real bad.
